lib/pages/sign_in/widgets/signin_form.dart

```diff
@@ -21,7 +21,7 @@ class _SignInFormState extends State<SignInForm> {
     final SignInBloc bloc = BlocProvider.of<SignInBloc>(context);
     return new Form(
       key: _formKey,
-        child: new Padding(
+      child: new Padding(
       padding: const EdgeInsets.symmetric(vertical: 0.0, horizontal: 15.0),
```

Comment:
Move padding values to one place (Like AppStyles or just Styles class)

Reply:
For this feature needs one more issue

Reply:
Fix in #21